endpoints/openrtb2/video_auction.go

```diff
@@ -510,10 +510,14 @@ func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.Bi
 
 func createBidExtension(videoRequest *openrtb_ext.BidRequestVideo) ([]byte, error) {
 
-	inclBrandCat := openrtb_ext.ExtIncludeBrandCategory{
-		PrimaryAdServer: videoRequest.IncludeBrandCategory.PrimaryAdserver,
-		Publisher:       videoRequest.IncludeBrandCategory.Publisher,
+	var inclBrandCat *openrtb_ext.ExtIncludeBrandCategory
+	if videoRequest.IncludeBrandCategory != nil {
+		inclBrandCat = &openrtb_ext.ExtIncludeBrandCategory{
```

Comment:
I wonder if we really need to copy here, can we just use "IncludeBrandCategory" type in ExtRequestTargeting struct so all you'd have to do here is say `inclBrandCat := videoRequest.IncludeBrandCategory`? If that would cause too many changes though this is fine as is, but it looks like "IncludeBrandCategory" and "ExtIncludeBrandCategory" are identical structures so not sure why have the "Ext" specific one.

Reply:
Actually disregard that previous comment, it may be that some point in the future the extension needs to change from the structure of the request so having them separate but identical structures isn't such a bad thing I suppose.